sgtool/file.go

```diff
@@ -55,6 +55,31 @@ func newFileState() *fileState {
 	}
 }
 
+func FromLocal(ctx context.Context, filepath string, opts ...Opt) error {
```

Comment:
Missing documentation here

Reply:
Sure. Added some comments on the function